src/gui/helpers/inventory_helper.rs

```diff
@@ -0,0 +1,29 @@
+use super::GuiHelper;
+use crate::state::GameState;
+use seq::prelude::*;
+
+pub const NAME: &str = "Inventory Helper";
+
+#[derive(Debug)]
+pub struct InventoryHelper;
+
+impl InventoryHelper {
+    pub fn create() -> Box<Self> {
+        Box::new(Self)
+    }
+}
+
+impl GuiHelper for InventoryHelper {
+    fn draw(
+        &mut self,
+        game_state: &GameState,
+        _sequencer: &mut Sequencer<GameState>,
+        ui: &mut egui::Ui,
+        _tab: &mut String,
+    ) {
+        let currency_manager = &game_state.memory_managers.currency_manager.data;
+
```

Comment:
Possibly mark this with a debug printout "more to be implemented" (assuming the actual inventory will go here later)